qcodes/tests/parameter/test_delegate_parameter.py

```diff
@@ -460,3 +459,20 @@ def test_delegate_parameter_fixed_label_unit_unchanged():
     delegate_param.source = None
     assert delegate_param.label == "delegatelabel"
     assert delegate_param.unit == "delegateunit"
+
+
+def test_cache_invalidation():
+    value = 10
+    p = BetterGettableParam('testparam', set_cmd=None, get_cmd=None)
+    d = DelegateParameter('test_delegate_parameter', p,
+                          initial_cache_value=value)
+    assert p._get_count == 0
+    assert d.cache.get() == value
+    assert p._get_count == 0
+    d.cache.invalidate()
+    p.cache.invalidate()
+    # since both the cache or the
+    # delegate and param is marked invalid
+    # this should trigger a get on the parameter
```

Comment:
shouldn't it trigger from a single invalidate() call on either of the parameters? delegate cache is just proxying to the soruce pararmater cache, right?